liquibase-standard/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -182,6 +184,16 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDefaultValue(false)
                 .build();
 
+        INCLUDE_CATALOG_NAME_FOR_DEFAULT = builder.define("includeCatalogNameForDefault", Boolean.class)
+                .setDescription("If true, the catalog name is included for the default schema")
```

Comment:
shouldn't it be `the catalog name is included for the default catalog` ?

Reply:
Thanks for pointing that out